libs/core/src/lib/button/Button.tsx

```diff
@@ -0,0 +1,39 @@
+import {
+  Pressable,
+  GestureResponderEvent,
+  StyleSheet,
+  Text,
+} from 'react-native';
+
+import { Palette } from 'bmg-news-app/src/app/constants/Palette';
+import { Typography } from 'bmg-news-app/src/app/constants/Typography';
```

Comment:
`Palette` და `Typography` ბიბლიოთეკიდან უნდა წამოვიღოთ.

```suggestion
import { Typography, Palette } from '@bmg-react-native-nx/constants';
```